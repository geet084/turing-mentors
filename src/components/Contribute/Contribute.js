import React from 'react';

export const Contribute = () => {
  return (
    <div className="link-content">
      <h2>Want to contribute?</h2>
      <p>Please send questions to:</p>
      <h4 className="dev-links">BE: <a href="#">Ricardo Ledesma</a></h4>
      <h4 className="dev-links">FE: <a href="#">Travis Gee</a></h4>
    </div>
  )
}

export default Contribute;