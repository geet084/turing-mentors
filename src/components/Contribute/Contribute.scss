.dev-links {
  a {
    margin: 20px 8px;
    font-size: .95rem;
    text-decoration: none;
    color: #5f5f5f;
    font-weight: 600;
    &:hover {
      margin-bottom: -3px;
      border-bottom: 3px solid #12f1fc;
      cursor: pointer;
    }
  }
}