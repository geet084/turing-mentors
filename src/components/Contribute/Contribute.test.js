import React from 'react';
import { shallow } from 'enzyme';
import { Contribute } from './Contribute';

describe('Contribute', () => {
  let wrapper;
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<Contribute />);
    expect(wrapper).toMatchSnapshot();
  });
});