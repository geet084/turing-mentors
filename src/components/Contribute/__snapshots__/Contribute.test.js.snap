// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contribute should have match the correct snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contribute />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Want to contribute?
        </h2>,
        <p>
          Please send questions to:
        </p>,
        <h4
          className="dev-links"
        >
          BE: 
          <a
            href="#"
          >
            Ricardo Ledesma
          </a>
        </h4>,
        <h4
          className="dev-links"
        >
          FE: 
          <a
            href="#"
          >
            Travis Gee
          </a>
        </h4>,
      ],
      "className": "link-content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Want to contribute?",
        },
        "ref": null,
        "rendered": "Want to contribute?",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Please send questions to:",
        },
        "ref": null,
        "rendered": "Please send questions to:",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "BE: ",
            <a
              href="#"
            >
              Ricardo Ledesma
            </a>,
          ],
          "className": "dev-links",
        },
        "ref": null,
        "rendered": Array [
          "BE: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ricardo Ledesma",
              "href": "#",
            },
            "ref": null,
            "rendered": "Ricardo Ledesma",
            "type": "a",
          },
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "FE: ",
            <a
              href="#"
            >
              Travis Gee
            </a>,
          ],
          "className": "dev-links",
        },
        "ref": null,
        "rendered": Array [
          "FE: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Travis Gee",
              "href": "#",
            },
            "ref": null,
            "rendered": "Travis Gee",
            "type": "a",
          },
        ],
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Want to contribute?
          </h2>,
          <p>
            Please send questions to:
          </p>,
          <h4
            className="dev-links"
          >
            BE: 
            <a
              href="#"
            >
              Ricardo Ledesma
            </a>
          </h4>,
          <h4
            className="dev-links"
          >
            FE: 
            <a
              href="#"
            >
              Travis Gee
            </a>
          </h4>,
        ],
        "className": "link-content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Want to contribute?",
          },
          "ref": null,
          "rendered": "Want to contribute?",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Please send questions to:",
          },
          "ref": null,
          "rendered": "Please send questions to:",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "BE: ",
              <a
                href="#"
              >
                Ricardo Ledesma
              </a>,
            ],
            "className": "dev-links",
          },
          "ref": null,
          "rendered": Array [
            "BE: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ricardo Ledesma",
                "href": "#",
              },
              "ref": null,
              "rendered": "Ricardo Ledesma",
              "type": "a",
            },
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "FE: ",
              <a
                href="#"
              >
                Travis Gee
              </a>,
            ],
            "className": "dev-links",
          },
          "ref": null,
          "rendered": Array [
            "FE: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Travis Gee",
                "href": "#",
              },
              "ref": null,
              "rendered": "Travis Gee",
              "type": "a",
            },
          ],
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
